docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+# 4. What Is An Operator?
+
+## TLDR:
+
+- Observables are a container for a collection of values over time.
```

Comment:
In Chapter 1 the text is: "An observable, in fact, is not a "stream". An observable is a template for subscription..." To be consistent with this definition, could this instead say "Observables *provide* a container..." or something similar instead of that they *are* a container?